code/api/src/migrations/6-order.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('orders', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'users',
+          key: 'id'
+        },
+        allowNull: false
+      },
+      crateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'crates',
+          key: 'id'
+        },
+        allowNull: false
+      },
+      deliveryDate: {
+        type: Sequelize.STRING
+      },
+      deliveryStatus: {
+        type: Sequelize.STRING
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('orders');
```

Comment:
Thanks for explaining that dropTable is to remove the entire crate (whereas removeColumn was for for removing one part of user.